ts/features/fci/saga/networking/handleGetSignatureRequestById.ts

```diff
@@ -0,0 +1,43 @@
+import { SagaIterator } from "redux-saga";
+import { call, put } from "typed-redux-saga/macro";
+import { ActionType } from "typesafe-actions";
+import * as E from "fp-ts/lib/Either";
+import { SagaCallReturnType } from "../../../../types/utils";
+import { readablePrivacyReport } from "../../../../utils/reporters";
+import { BackendFciClient } from "../../api/backendFci";
+import { fciSignatureRequestFromId } from "../../store/actions";
+import { getNetworkError } from "../../../../utils/errors";
+
+// handle bonus list loading
+export function* handleGetSignatureRequestById(
+  getSignatureDetailViewById: ReturnType<
+    typeof BackendFciClient
+  >["getSignatureDetailViewById"],
+  action: ActionType<typeof fciSignatureRequestFromId["request"]>
+): SagaIterator {
+  try {
+    const getSignatureDetailViewByIdResponse: SagaCallReturnType<
+      typeof getSignatureDetailViewById
+    > = yield* call(getSignatureDetailViewById, { id: action.payload });
+    if (E.isLeft(getSignatureDetailViewByIdResponse)) {
```

Comment:
Can we avoid using **if** (i know it is the old style already used in the app)?

Reply:
Could you provide me a reference for the "new style" used in the app?